src/main/resources/export/cpp/ExportDescription.yaml

```diff
@@ -132,143 +134,157 @@ Instructions:
     Construction: "#variable($Name).reset(new frc::${ClassName}(*#variable($Left_Motor), *#variable($Right_Motor)));"
-    LiveWindow: "AddChild(\"$Short_Name\", #variable($Name));"
+    LiveWindow: "AddChild(\"$Short_Name\", &m_#variable($Name));"
     Import: "\\#include \"frc/drive/${ClassName}.h\""
     Extra: >
-      #variable($Name)->SetSafetyEnabled($Safety_Enabled);
-              #variable($Name)->SetExpiration($Safety_Expiration_Time);
-              #variable($Name)->SetMaxOutput($Maximum_Output);
+      m_#variable($Name).SetSafetyEnabled($Safety_Enabled);
+              m_#variable($Name).SetExpiration($Safety_Expiration_Time);
+              m_#variable($Name).SetMaxOutput($Maximum_Output);
   Mecanum Drive:
     Defaults: "Component,None"
     ClassName: "MecanumDrive"
-    LiveWindow: "AddChild(\"$Short_Name\", #variable($Name));"
+    Declaration: "frc::${ClassName} m_#variable($Name){m_#variable($Left_Front_Motor), m_#variable($Left_Rear_Motor),
+                    m_#variable($Right_Front_Motor), m_#variable($Right_Rear_Motor)};"
+    LiveWindow: "AddChild(\"$Short_Name\", &m_#variable($Name));"
     Import: "\\#include \"frc/drive/${ClassName}.h\""
     Construction: >-
       #variable($Name).reset(new frc::${ClassName}(*#variable($Left_Front_Motor), *#variable($Left_Rear_Motor),
                     *#variable($Right_Front_Motor), *#variable($Right_Rear_Motor)));
     Extra: >
-      #variable($Name)->SetSafetyEnabled($Safety_Enabled);
-              #variable($Name)->SetExpiration($Safety_Expiration_Time);
-              #variable($Name)->SetMaxOutput($Maximum_Output);
+      m_#variable($Name).SetSafetyEnabled($Safety_Enabled);
+              m_#variable($Name).SetExpiration($Safety_Expiration_Time);
+              m_#variable($Name).SetMaxOutput($Maximum_Output);
   Killough Drive:
     Defaults: "Component,None"
     ClassName: "KilloughDrive"
-    LiveWindow: "AddChild(\"$Short_Name\", #variable($Name));"
+    Declaration: "frc::${ClassName} m_#variable($Name){m_#variable($Left_Motor), m_#variable($Right_Motor),
+                    m_#variable($Back_Motor)};"
+    LiveWindow: "AddChild(\"$Short_Name\", &m_#variable($Name));"
     Import: "\\#include \"frc/drive/${ClassName}.h\""
     Construction: >-
       #variable($Name).reset(new frc::${ClassName}(*#variable($Left_Motor), *#variable($Right_Motor),
                     *#variable($Back_Motor)));
     Extra: >
-      #variable($Name)->SetSafetyEnabled($Safety_Enabled);
-              #variable($Name)->SetExpiration($Safety_Expiration_Time);
-              #variable($Name)->SetMaxOutput($Maximum_Output);
+      m_#variable($Name).SetSafetyEnabled($Safety_Enabled);
+              m_#variable($Name).SetExpiration($Safety_Expiration_Time);
+              m_#variable($Name).SetMaxOutput($Maximum_Output);
 
   AnalogGyro:
     Defaults: "AnalogInput,Component,None"
     ClassName: "AnalogGyro"
     Extra: "#variable($Name)->SetSensitivity(${Sensitivity});"
```

Comment:
```suggestion
    Extra: "#variable($Name).SetSensitivity(${Sensitivity});"
```

Reply:
the gearsbot example shows a "." instead of a "->" and when i try to use "->" it throws an error about the expression not being pointer type.
if the "." vs "->" distinction is importnat than the examples should probably be changed as well.

Reply:
The suggestion is to change it from -> to a . -> is used when its a pointer, . is used when it's not.